tests/test_miner_data_handler.py

```diff
@@ -48,15 +50,22 @@ def test_get_values_exceeding_range(self):
                                                 2024-11-30T00:00:00
                                                         |-|        (Current Time - more than 5 days passed)
         """
-        miner_id = "miner_123"
+        miner_id = 1
         start_time = "2024-11-20T00:00:00"
         current_time = "2024-11-30T00:00:00"
+        simulation_input = SimulationInput(
+            asset="BTC",
+            start_time=start_time,
+            time_increment=300,
+            time_length=86400,
+            num_simulations=100
+        )
 
         values = generate_values(datetime.fromisoformat(start_time))
-        self.handler.set_values(miner_id, start_time, values)
+        self.handler.set_values(miner_id, values, simulation_input)
 
         result = self.handler.get_values(miner_id, current_time)
-        self.assertEqual(result, [])
+        # self.assertEqual(result, []) # TODO: do we want this 5 days expiry?
```

Comment:
@parkhomenko

Reply:
yes, let's leave this todo, I think we need some expiration time, there are a lot of data going to be in the database because of the predictions size, and we'll need to remove the older records from the database as well, this is something we still need to think about:

now I see that the current prediction for a day for 1 miner consumes space in the db of 200KB,
we run validations every hour and we can potentially have 200 miners

in total, if we run the validator for a single week this results in 672GB of data